src/collectd.conf.pod

```diff
@@ -328,6 +328,51 @@ the daemon should try to figure out the "fully qualified domain name", FQDN.
 This is done using a lookup of the name returned by C<gethostname>. This option
 is enabled by default.
 
+=item B<Resource> I<Type>
+
+The B<Resource> block defined the I<Resource Attributes> to use for local
+metrics such as CPU usage, i.e. metrics that were not received from a remote
```

Comment:
Reads better to me when tenses are switched:
```suggestion
B<Resource> block defines I<Resource Attributes> to be used for local
metrics such as CPU usage, i.e. metrics that were not received from a remote
```

Reply:
Fixed the typo. I prefer the active voice though.